facebook/draft-js
# fix(selection): Returns previous selection if either leaf is null

**Summary**

Bails out early from the `getUpdatedSelectionState` method with previous selection when `anchorLeaf` or `focusLeaf` are null to prevent an error surfaced when using a decorator for emoji inserted between two spans of text entered via IME.

Thanks to @robbertbrak for suggesting and cherry-picking a fix from their fork 🎉 

Fixes #2187

**Test Plan**
1. Build Draft.js
1. Inject into jsfiddle reported by @jdecked https://jsfiddle.net/36tvhmce/
1. Try to repro issues with Pinyin and Hiragana IME
1. Verify there is no error in the console and text is entered normally
